install_files/ansible-base/securedrop-prod.yml

```diff
@@ -2,6 +2,8 @@
 ---
 - name: Ensure validation is run before prod install
   hosts: localhost
+  environment:
+    LC_ALL: C
```

Comment:
What do you think about validating that this variable is set (via `assert` or otherwise) at the start of each play?

Reply:
Rather than an assert statement, we can add a config test (to be run in CI) that confirms the environment is set as expected _on every play_. See here for an example: https://github.com/freedomofpress/securedrop/blob/10a2eeedc3c22fd39e6a7eda4ca8f4bca1ca6024/molecule/ansible-config/tests/test_max_fail_percentage.py

Reply:
@rmol has already added the corresponding config test here, which will help guard against regressions in the future: if new plays are added _without_ the explicit locale declaration, CI will report an error.